src/main/java/frc/robot/subsystems/drivetrain/TrigonDrive.java

```diff
@@ -0,0 +1,65 @@
+package frc.robot.subsystems.drivetrain;
+
+import edu.wpi.first.wpilibj.SpeedController;
+import edu.wpi.first.wpilibj.drive.DifferentialDrive;
+
+import static frc.robot.Robot.robotConstants;
+
+/**
+ * This class includes all the methods from differential drive plus a few of our
+ * own based on our drivers request
+ */
+// TODO: add documentation
+public class TrigonDrive extends DifferentialDrive {
+
+    private double sensitivity;
+    private double threshold;
+
+    public TrigonDrive(SpeedController leftMotor, SpeedController rightMotor) {
+        this(leftMotor, rightMotor,
+                robotConstants.trigonDriveConstents.SENSITIVITY, robotConstants.trigonDriveConstents.THRESHOLD);
+    }
+
+    public TrigonDrive(SpeedController leftMotor, SpeedController rightMotor, double sensitivity,
+            double threshold) {
+        super(leftMotor, rightMotor);
+        this.sensitivity = sensitivity;
+        this.threshold = threshold;
+    }
+
+    public void setSensitivity(double sensitivity) {
+        this.sensitivity = sensitivity;
+    }
+
+    public void setThreshold(double threshold) {
+        this.threshold = threshold;
+    }
+
+    public double getSensitivity() {
+        return sensitivity;
+    }
+
+    public double getThreshold() {
+        return threshold;
+    }
+
+    // public double yInputCalculation(double value) {
+    //     boolean isLinear = Math.abs(value) <= 0.25;
+    //     return isLinear ? 2 * value : Math.signum(value) * Math.sqrt(Math.abs(value));
```

Comment:
0.25 to class final Y_LINEAR_THRESHOLD
2 to class final y_linear_coefficient (all caps)